public/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "shelf-webapp",
+  "lang": "en-US",
+  "short_name": "shelf",
+  "description": "Modern Open Source Asset Management System",
+  "start_url": "/",
+  "display": "standalone",
+  "scope": "/",
+  "theme_color": "#EF6820",
+  "background_color": "#FFFFFF",
+  "icons": [
```

Comment:
We need to add more icons as this is not really good like this. For example when I added it on my iphone to the home screen, because of the wrong icon format we have a white border on ios for example. I used this to generate the assets: https://progressier.com/pwa-icons-and-ios-splash-screen-generator

Here is the output: 
[splash_screens.zip](https://github.com/Shelf-nu/shelf.nu/files/12868498/splash_screens.zip)

Reply:
Cool, adding these splash screens. Also, we need to setup different size icons, as of now I added only x1024.

Reply:
Okey according to the site i checked, it should already give the icon. I can see there is an icon file that is 512x512. Can you try replacing the one you had added with it? Then i can test it again on my local.